update-pr-branch-version/action.yml

```diff
@@ -20,9 +20,9 @@ runs:
   using: composite
   steps:
     - name: Python Poetry Action
-      uses: abatilo/actions-poetry@v2.1.6
+      run: pipx install poetry==1.1.15
 
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
```

Comment:
The current [GH checkout action](https://github.com/actions/checkout) is at 4.2.1. Why bump to 3 and not 4?

Reply:
I'll bump to 4, thanks. I had 3 in there as I know as of last week, v3 wasnt breaking any of our stuff :)